CodingChallenges/CC_130_Fourier_Transform_1/Processing/CC_130_Fourier_Transform/CC_130_Fourier_Transform.pde

```diff
@@ -0,0 +1,96 @@
+// Coding Train
+// Ported to processing by Max (https://github.com/TheLastDestroyer)
+// Origional JS by Daniel Shiffman
```

Comment:
```suggestion
// Original JS by Daniel Shiffman
```

Reply:
I've fixed it in my branch but it's not coming through to the new pr, not sure if (or how) I can change it here